master/src/main/java/org/apache/celeborn/service/deploy/master/clustermeta/ha/HARaftServer.java

```diff
@@ -163,6 +166,8 @@ public static HARaftServer newMasterRatisServer(
             .setId(localRaftPeerId)
             .setAddress(ratisAddr)
             .setClientAddress(localNode.rpcEndpoint())
+            // We use admin address to host the internal rpc address
+            .setAdminAddress(localNode.internalRpcEndpoint())
```

Comment:
Note to reviewers - I am using `setAdminAddress` as a mechanism to store the internal rpc endpoint - I am not very familiar with ratis, please do validate if this is fine.

Reply:
I think it's OK, maybe @SzyWilliam can help to confirm